reo/validators.py

```diff
@@ -1459,15 +1502,17 @@ def fillin_defaults(self, object_name_path, template_values=None, real_values=No
                     if input_isDict or input_isDict is None:
                         self.defaults_inserted.append([template_key, object_name_path])
                     if input_isDict is False:
-                        object_name_path[-1] = object_name_path[-1] + ' (number {})'.format(number)
+                        if not object_name_path[-1].endswith(' (number {})'.format(number)):
```

Comment:
Is this just for multiple PV's? I find the `number` name confusing every time I look at the validator. It is an index of the multiple PV's right?

Reply:
Right now only PV is allowed to be a list or dict, so yes. In the future this is extensible to other inputs like this